CMakeLists.txt

```diff
@@ -172,8 +171,30 @@ include(ReleaseDebugAutoFlags)
 find_package(BISON REQUIRED)
 find_package(FLEX REQUIRED)
 
-if(NOT NRN_ENABLE_INTERNAL_READLINE)
-  find_package(Readline)
+# If NEURON_WHEEL_BUILD is set, look for static libraries first.
+# NEURON_WHEEL_BUILD is set for wheel building.
```

Comment:
I prefer cmake option than env variables for build system. Sometime we set env variable and forget about them. Being explicit via cmake option is better?

Reply:
No preference on my side, we can go for a CMake option.

Reply:
discussed offline: To make sure 100% we have the proper MACOSX_DEPLOYMENT_TARGET, we should build `readline` and `ncurses` and have them available somewhere.